egs/wsj/s5/steps/dur_model/get_examples.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+cmd=run.pl
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 2 ]; then
+   echo "Usage: $0 [options] <ali-dir> <duration-model-dir>"
+   echo "e.g.: $0 exp/mono_ali exp/mono/durmod"
+   exit 1;
+fi
+
+alidir=$1
+dir=$2
+
+durmodel=$dir/durmodel.mdl
+transmodel=$alidir/final.mdl
+
+for f in $transmodel $alidir/ali.1.gz; do
+  [ ! -f $f ] && echo "$0: Required file not found: $f" && exit 1;
+done
+
+if [ ! -f $dir/all.ali ]; then
+  gunzip -c $alidir/ali.*.gz >> $dir/all.ali || exit 1;
+fi
+
+if [ ! -f $dir/all.egs ]; then
+$cmd $dir/log/durmod_make_examples.log \
+     nnet3-durmodel-make-egs $durmodel $transmodel ark:$dir/all.ali ark,t:$dir/all.egs || exit 1;
+fi
+
+numparts=20
```

Comment:
>
> > +if [ ! -f $dir/all.egs ]; then
>
> > +$cmd $dir/log/durmod_make_examples.log \
> > +     nnet3-durmodel-make-egs $durmodel $transmodel ark:$dir/all.ali ark,t:$dir/all.egs || exit 1;
> > +fi
> > +
> > +numparts=20
>
> Therefore, the best WER that is reported using best_wer.sh (after checking
> several lm-scales) is not completely valid either. Is that right?
>
Yes, in a sense- in theory one should get the scales from a dev set.


> How about Switchboard? You could just run it with a GMM system, since it's
> independent of the model type. Make a directory in /export/[something],
> instead of using your home directory.
>
> OK, thank you. I currently have a compiled kaldi at my home directory.
> Should I move it from the home directory?
>
Yes, best to move it.  E.g. to /export/a10.
Dan



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/414/files#r49952004>.
>